app/javascript/components/action-form/helper.js

```diff
@@ -0,0 +1,103 @@
+const dataHelper = (values) => {
+  const vd = values.description;
+  const vt = values.action_type;
+  const optionValue = values.options;
+  delete values.action_type;
+  delete values.description;
+  let data = {};
+
+  if (vt === 'delete_snapshots_by_age') {
```

Comment:
Is it possible to write this logic(if vt=== ...) with a switch statement for cleaner code?